lms/djangoapps/bulk_email/tasks.py

```diff
@@ -408,6 +408,11 @@ def _get_source_address(course_id, course_title):
         re.sub(r"[^\w.-]", '_', course_id.course),
         settings.BULK_EMAIL_DEFAULT_FROM_EMAIL
     )
+
+    # make sure it's not longer than 320 chars
+    if len(from_addr) >= 320:
```

Comment:
@cahrens oh, interesting! I think I do know how to check this — setting a RTL course display name should do the trick.